bot/services/liked_tweet.py

```diff
@@ -33,6 +33,41 @@ def get_liked_tweets(user_id: str) -> list[tweepy.Tweet]:
     return all_tweets
 
 
+def get_liked_tweets_since(user_id: str, since: int | None = None):
+    all_tweets: list[tweepy.Tweet] = []
+    next_token = None
+
+    for _ in count(1):
+        tweets = retry_function_if_fail(
+            TwitterClient.client.get_liked_tweets,
+            id=user_id,
+            tweet_fields=FetchConfigs.tweet_fields,
+            max_results=5,
+            pagination_token=next_token,
+        )
+        tweets_list = tweets.data
+        tweets_meta = tweets.meta
+
+        tweets_list = tweets_list if tweets_list is not None else []
+        all_tweets += tweets_list
+
+        if since:  # check if there is a tweet that was created before the `since`
+            tweets_time = list(
+                map(lambda tweet: tweet.created_at.timestamp() * 1000, tweets_list)
+            )
+            times_before_since = list(filter(lambda time: time < since, tweets_time))
+            if len(times_before_since) > 1:
+                return all_tweets
```

Comment:
Suggestion: Having multiple return statements can reduce code readability, So I'd suggest here to just break the loop and the return would happen at line 68.